routing/routing_tests/turns_sound_test.cpp

```diff
@@ -357,7 +357,7 @@ UNIT_TEST(TurnsSound_ComposedTurnTest)
   vector<TurnItemDist> const turns2 = {{{5 /* idx */, CarDirection::TurnRight}, 620. /* m_distMeters */},
                                        {{10 /* idx */, CarDirection::EnterRoundAbout}, 665. /* m_distMeters */}};
   vector<string> const expectedNotification2 = {{"In 600 meters. Turn right."},
-                                                {"Then. Enter the roundabout."}};
+                                                {"Then.  Enter the roundabout."}};
```

Comment:
The extra space is where "In 500 feet" would go if it was going to be pronounced, like `{then} {distance} {direction}` but since this set of instructions does not have a distance it ends up being `then (null) enter the roundabout`. Rather than add more string manipulation logic to the TTS engine that will not affect output but only increase complexity and computation, I adjusted the test to validate that this is indeed an acceptable TTS output. I could also add whitespace collapsing logic to the tests, but it seemed superfluous.

Reply:
@biodranik I've removed extra spaces (some of which existed before this PR) by adding more logic to the TTS engine